csv2rdf/index.html

```diff
@@ -893,13 +778,15 @@
                   the <code>name</code> property specified for each
                   <a href="http://www.w3.org/TR/tabular-data-model/#dfn-column">column</a>. 
                   During template expansion, the <em>variables</em> evaluate to the 
-                  <a href="http://www.w3.org/TR/tabular-data-model/#dfn-string-value">cell value</a>
+                  <a href="http://w3c.github.io/csvw/metadata/#dfn-cell-value">cell value</a>
                   within the 
                   <a href="http://www.w3.org/TR/tabular-data-model/#dfn-row">row</a>
                   being processed that is associated with the named 
                   <a href="http://www.w3.org/TR/tabular-data-model/#dfn-column">column</a>.</p>
                 <p>The <em>variable</em> <code>_row</code> evaluates to the number of the row being
```

Comment:
My own implementation maintains a `_raw` value for the Cell, but it's not used anywhere. The _cell value_ should include the effect cell value(s) after transformation, including the `valueUrl` expanded values.

In this code the _cell value_ reference should get this bit from the Metadata document:

> The cell value of a cell is the result processing the cell as defined here and applying any format-specific mappings to create on or more values. Converters must be provided these cell values, along with any in-scope lang and datatype property values.

> The process of parsing the string value of a cell into a single value or a list of values is as follows:

> * unless the valueUrl is defined on the cell, datatype is string or anySimpleType or any, strip leading and trailing whitespace from the value
> * if the value is the same as the null value, then the value is null
> * if the separator property is not null, create a list of values by splitting the string at the character specified by the separator property
> * validate the value(s) against the format, if one is specified, as described below; raise an error if any of the values do not match the specified format
> * if the cell column has a valueUrl the value(s) are the result of applying the URI template to each value as a resource
> * parse the value(s) using the format, as described below

(Note, the use of `valueUrl` should probably strip whitespace, in spite of what this says.)

The Transformation documents can then just reference _cell value_ to get what the need. If there's a requirement to still have the raw value available, that should probably be a separate issue.